### ISSUE TITLE
Do not disable zooming

### ISSUE THREAD

**Issue by njaffer** [mlibrary/umrdr#555]
On homepage, deposit work, My Works, My Collection, Dashboard

**Comment by respinos** [comment]
`<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">`  should be `<meta name="viewport" content="width=device-width, initial-scale=1.0">`, via http://a11yproject.com/posts/never-use-maximum-scale/
